Converting a value to empty interface makes yaegi panic.

The following program `main.go` triggers a panic:

```go
package main

import "fmt"

func main() {
	b := 0

	v := interface{}(b)

	fmt.Println(v)
}
```

Expected result:
```console
$  go run main.go
0
```

Got:
```console
$ yaegi main.go 
main.go:8:7: CFG post-order panic: runtime error: invalid memory address or nil pointer dereference
goroutine 1 [running]:
runtime/debug.Stack(0x1, 0xc000396000, 0x40)
	/usr/lib/go/src/runtime/debug/stack.go:24 +0x9d
github.com/containous/yaegi/interp.(*Interpreter).Eval.func1(0xc0001addf0)
	/home/jlevesy/code/pkg/mod/github.com/containous/yaegi@v0.8.6/interp/interp.go:320 +0xbc
panic(0xd796a0, 0xc000204680)
	/usr/lib/go/src/runtime/panic.go:969 +0x166
github.com/containous/yaegi/interp.(*Interpreter).cfg.func2.1(0xc000381000)
	/home/jlevesy/code/pkg/mod/github.com/containous/yaegi@v0.8.6/interp/cfg.go:400 +0xb3
panic(0xce2960, 0x1548e70)
	/usr/lib/go/src/runtime/panic.go:969 +0x166
github.com/containous/yaegi/interp.(*itype).refType(0x0, 0xc0001ad060, 0x1567a00, 0x7f709d8bc560, 0x0)
	/home/jlevesy/code/pkg/mod/github.com/containous/yaegi@v0.8.6/interp/type.go:1071 +0x3a
github.com/containous/yaegi/interp.(*itype).TypeOf(0x0, 0xc000373420, 0x1)
	/home/jlevesy/code/pkg/mod/github.com/containous/yaegi@v0.8.6/interp/type.go:1164 +0x96
github.com/containous/yaegi/interp.(*Interpreter).cfg.func2(0xc000381000)
	/home/jlevesy/code/pkg/mod/github.com/containous/yaegi@v0.8.6/interp/cfg.go:800 +0xa4c6
github.com/containous/yaegi/interp.(*node).Walk(0xc000381000, 0xc0001adc98, 0xc0001adc68)
	/home/jlevesy/code/pkg/mod/github.com/containous/yaegi@v0.8.6/interp/interp.go:191 +0xaf
github.com/containous/yaegi/interp.(*node).Walk(0xc000380e00, 0xc0001adc98, 0xc0001adc68)
	/home/jlevesy/code/pkg/mod/github.com/containous/yaegi@v0.8.6/interp/interp.go:188 +0x70
github.com/containous/yaegi/interp.(*node).Walk(0xc000380a00, 0xc0001adc98, 0xc0001adc68)
	/home/jlevesy/code/pkg/mod/github.com/containous/yaegi@v0.8.6/interp/interp.go:188 +0x70
github.com/containous/yaegi/interp.(*node).Walk(0xc000380500, 0xc0001adc98, 0xc0001adc68)
	/home/jlevesy/code/pkg/mod/github.com/containous/yaegi@v0.8.6/interp/interp.go:188 +0x70
github.com/containous/yaegi/interp.(*node).Walk(0xc000380000, 0xc0001adc98, 0xc0001adc68)
	/home/jlevesy/code/pkg/mod/github.com/containous/yaegi@v0.8.6/interp/interp.go:188 +0x70
github.com/containous/yaegi/interp.(*Interpreter).cfg(0xc000370000, 0xc000380000, 0x7ffd3ddb518c, 0x7, 0xc000212068, 0x4, 0x7ffd3ddb518c, 0x7, 0x0)
	/home/jlevesy/code/pkg/mod/github.com/containous/yaegi@v0.8.6/interp/cfg.go:55 +0x160
github.com/containous/yaegi/interp.(*Interpreter).Eval(0xc000370000, 0xc00037a000, 0x5b, 0x0, 0x0, 0x0, 0x0, 0x0)
	/home/jlevesy/code/pkg/mod/github.com/containous/yaegi@v0.8.6/interp/interp.go:343 +0x1ea
main.main()
	/home/jlevesy/code/pkg/mod/github.com/containous/yaegi@v0.8.6/cmd/yaegi/yaegi.go:138 +0x553
```

--- a/_test/interface42.go
+++ b/_test/interface42.go
@@ -1,0 +1,12 @@
+package main
+
+import "fmt"
+
+func main() {
+	v := interface{}(0)
+	
+	fmt.Println(v)
+}
+
+// Output:
+// 0

--- a/_test/interface42.go
+++ b/_test/interface42.go
@@ -4,7 +4,7 @@ import "fmt"
 
 func main() {
 	v := interface{}(0)
-	
+
 	fmt.Println(v)
 }
 

--- a/interp/cfg.go
+++ b/interp/cfg.go
@@ -388,7 +388,8 @@ func (interp *Interpreter) cfg(root *node, pkgID string) ([]*node, error) {
 			return false
 
 		case interfaceType:
-			n.typ = sc.getType("interface{}")
+			//n.typ = sc.getType("interface{}")
+			n.typ, err = nodeType(interp, sc, n)
 			return false
 		}
 		return true

--- a/interp/cfg.go
+++ b/interp/cfg.go
@@ -383,12 +383,7 @@ func (interp *Interpreter) cfg(root *node, pkgID string) ([]*node, error) {
 				}
 			}
 
-		case arrayType, basicLit, chanType, funcType, mapType, structType:
-			n.typ, err = nodeType(interp, sc, n)
-			return false
-
-		case interfaceType:
-			//n.typ = sc.getType("interface{}")
+		case arrayType, basicLit, chanType, funcType, interfaceType, mapType, structType:
 			n.typ, err = nodeType(interp, sc, n)
 			return false
 		}

--- a/interp/cfg.go
+++ b/interp/cfg.go
@@ -386,6 +386,10 @@ func (interp *Interpreter) cfg(root *node, pkgID string) ([]*node, error) {
 		case arrayType, basicLit, chanType, funcType, mapType, structType:
 			n.typ, err = nodeType(interp, sc, n)
 			return false
+
+		case interfaceType:
+			n.typ = sc.getType("interface{}")
+			return false
 		}
 		return true
 	}, func(n *node) {

